### DISCUSSION TITLE
How to read "Temp_Winding_Max" variable values.?

### DISCUSSION THREAD

**Discussion by Naveen-Kumar-B** [ansys/pymotorcad#367] [Q&A]
Here is the syntax, to read values. But it was not performing well
> Dim WindingTemp
> Call mcad.GetVariable("Temp_Winding_Maximum", WindingTemp)

@jgsdavies can i get help here.?

**Comment by jgsdavies** [comment]
Hi @B-Naveen-Kumar. This doesn't look like Python/PyMotorCAD so not sure I am able to help with this

**Reply by Naveen-Kumar-B** [reply]
Hi @jgsdavies , Its a VB scirpt.

Mean while pls share your mail id so that  i can share u the complete script..!!

**Reply by ravi-ansys** [reply]
Hello @B-Naveen-Kumar  As this is a VB script issue may you please visit https://customer.ansys.com and create a new Case and a support Engineer will contact you.

**Reply by Naveen-Kumar-B** [reply]
Hi @ravi-ansys , 
Thank you for your response.

**Reply by Naveen-Kumar-B** [reply]
> Hello @B-Naveen-Kumar As this is a VB script issue may you please visit https://customer.ansys.com and create a new Case and a support Engineer will contact you.

Dear @ravi-ansys , the above mentioned link is not working.!!

**Reply by ravi-ansys** [reply]
@B-Naveen-Kumar  my mistake may be. it seems the link is for customers however you belong to a channel partner account? I guess in this case you have a portal to raise a ticket ( ie salesforce)

**Reply by Naveen-Kumar-B** [reply]
@ravi-ansys Thanks
